src/modules/client/pages/auxiliaries/planning/AuxiliaryAgenda.vue

```diff
@@ -112,6 +113,11 @@ export default {
     await Promise.all([this.getAuxiliaries(), this.getCustomers(), this.refresh(), this.setInternalHours()]);
   },
   methods: {
+    setEvent (payload) {
+      const { path, value } = payload;
+      if (this.creationModal) set(this.newEvent, `${path}`, value);
```

Comment:
tu ne peux pas mettre directement `set(this.newEvent, path, value);` ?

Reply:
Si bien vu, je corrige :)